src/hooks/spoofing/write_physical_script_game_state_data_node.cpp

```diff
@@ -0,0 +1,29 @@
+#include "hooking/hooking.hpp"
+#include "util/model_info.hpp"
+
+#include <netsync/nodes/physical/CPhysicalScriptGameStateDataNode.hpp>
+
+namespace big
+{
+	void hooks::write_physical_script_game_state_data_node(__int64 iface, CPhysicalScriptGameStateDataNode* node)
+	{
+		g_hooking->get_original<hooks::write_physical_script_game_state_data_node>()(iface, node);
+
+		auto entity = reinterpret_cast<CVehicle*>((((rage::netObject*)(iface - 0x130)))->GetGameObject());
```

Comment:
Actually, if you change iface to CVehicle, you should be able to just use that instead of doing this hacky kinda weird way of grabbing the root object from the 2nd parameter.